tests/manual-test-examples/p5vr/basic/example.js

```diff
@@ -18,6 +18,7 @@ function calculate() {
 }
 
 function draw() {
+  setViewerPosition(0, 0, 20);
```

Comment:
Would be better to move the object rather than viewer to keep these examples conceptually simple